src/api/BdmsContextExtensions.cs

```diff
@@ -372,19 +369,20 @@ int GetStratigraphyOrCasingId(int currentLayerId, int startId)
 
         Layer SeededLayers(int seed) => fakelayers.UseSeed(seed).Generate();
 
+        var layersToInsert = new List<Layer>();
         for (int i = 0; i < stratigraphyRange.Count; i++)
         {
             // Add 10 layers per stratigraphy
             var start = (i * 10) + 1;
             var range = Enumerable.Range(start, 10);  // ints in range must be different on each loop, so that properties are not repeated in dataset.
-            context.Layers.AddRange(range.Select(SeededLayers));
+            layersToInsert.AddRange(range.Select(SeededLayers));
         }
 
-        context.SaveChanges();
+        context.BulkInsert(layersToInsert, bulkConfig);
 
         // Seed workflows
         var workflow_ids = 5000;
-        var workflowRange = Enumerable.Range(workflow_ids, 200);
+        var workflowRange = Enumerable.Range(workflow_ids, boreholeRange.Count);
```

Comment:
PP: Ich denke, wenn wir ohnehin nur 1 Workflow pro Boreholes haben, wäre es sinnvoll den so zu konfigurieren, dass das Borehole immer Change in Progress und durch Admin editierbar ist. Das war ja bislang nicht der Fall und zt etwas mühsam zu testen.  Dazu reicht es glaube ich sowohl die Rolle als auch den user auf 1 zu setzen und bei Finished nichts einzutragen.